scripts/kmod-symlink.sh

```diff
@@ -7,4 +7,9 @@ set -euo pipefail
 #
 # For context read through https://github.com/mesonbuild/meson/issues/9
 
-ln -sf kmod "$MESON_INSTALL_DESTDIR_PREFIX/$1"
+if [[ $1 == /* ]]; then
+    echo "Absolute destination folder provided, ignoring prefix"
+    ln -sf kmod "$DESTDIR/$1"
```

Comment:
https://github.com/kmod-project/kmod/blob/33e3d24cdb270cdfab71e9bd1c856e6430b3c11e/meson.build#L445-L446

join_paths is already abspath aware -- just move path building for PREFIX into meson.build

Note recent versions of meson can natively install symlinks.

Reply:
> join_paths is already abspath aware -- just move path building for PREFIX into meson.build

Update to always use prepend prefix. Thanks o/

> Note recent versions of meson can natively install symlinks.

That would be awesome, do you have any references?

Reply:
https://mesonbuild.com/Reference-manual_functions.html#install_symlink

You would need to bump the minimum required version from 0.60 to 0.61 but I'm not aware of any situation where that is a hardship, e.g. updating that version still fits within the last version to support python 3.6 (which was 0.61) and there are no distros that ship with exactly meson 0.60 other than Fedora 36 which feels a bit overly specific.